models/cluster.go

```diff
@@ -1,17 +1,21 @@
 package models
 
-//Type of cluster types for starter cluster.
-type StarterClusterType string
+// ClusterType represents the type of the cluster.
+type ClusterType string
 
 const (
-	// This type represents Free Cluster.
-	Free StarterClusterType = "FREE"
-	// This type represents Small Cluster.
-	Small StarterClusterType = "SMALL"
-	// This type represents Medium Cluster.
-	Medium StarterClusterType = "MEDIUM"
-	// This type represents Large Cluster.
-	Large StarterClusterType = "LARGE"
+	// Free cluster type.
+	Free ClusterType = "FREE"
+	// Small cluster type.
+	Small ClusterType = "SMALL"
+	// Medium cluster type.
+	Medium ClusterType = "MEDIUM"
+	// Large cluster type.
+	Large ClusterType = "LARGE"
+	// Serverless cluster type.
+	Serverless ClusterType = "SERVERLESS"
+	// Devmode cluster type.
+	Devmode ClusterType = "DEVMODE"
```

Comment:
I think DEVMODE is a subclass of SERVERLESS and not really a separate cluster type - to be confirmed tomorrow

Reply:
In our current implementation we have a separate cluster type: `DEV_MODE`